services/views/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "pins-views",
+  "version": "1.0.0",
+  "description": "nunjucks views used by pins",
+  "main": "index.js",
+  "scripts": {
+    "lint": "eslint ./",
+    "lint:fix": "eslint ./ --fix",
+    "format": "prettier --write './**/*.js'",
+    "test": "jest --coverage --maxWorkers=4"
+  },
+  "dependencies": {
+    "govuk-frontend": "^3.13.0",
+    "pino": "^6.7.0"
```

Comment:
It doesn't look like we're doing any logging so do we need pino as a dependency?

Reply:
Removed